azure-quantum/azure/quantum/target/microsoft/elements/dft/__init__.py

```diff
@@ -2,5 +2,6 @@
 
 from .target import MicrosoftElementsDft
 from .job import MicrosoftElementsDftJob
+from .libqcschema import *
```

Comment:
Kirill was mentioning that this might break customers who do not want to do the conversion and do not have pyscf installed. It might be better to put libqcshema in a separate namespace.